include/partition/inertial_flow.hpp

```diff
@@ -2,20 +2,56 @@
 #define OSRM_PARTITION_INERTIAL_FLOW_HPP_
 
 #include "partition/graph_view.hpp"
+
+#include <unordered_set>
 #include <vector>
 
 namespace osrm
 {
 namespace partition
 {
 
+// TODO: dummy
+
+struct MinCut
+{
+    std::int32_t num_edges;
+    // Should actually contain cut edges and/or lhs and rhs nodes
+};
+
+inline MinCut Dinic(const GraphView &graph,
+                    const std::unordered_set<NodeID> &sources,
+                    const std::unordered_set<NodeID> &sinks)
+{
+    return MinCut{10};
+}
+
+//
```

Comment:
Please rip out and adapt when merging. This is really just a dummy impl. for me to plug in.